migrator-tool/src/test/scala/za/co/absa/spline/migrator/DataLineageToExecPlanWithEventConverterSpec.scala

```diff
@@ -299,19 +299,20 @@ object DataLineageToExecPlanWithEventConverterSpec {
           childIds = Seq(7),
           schema = None,
           params = Map(
-            "name" -> "alias",
-            "alias" -> "my-df")),
+            //todo: https://github.com/AbsaOSS/spline/issues/496
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Take the required action to fix the issue indicated by this comment.](https://app.codacy.com/app/ABSA_OSS/spline/pullRequest?prid=4594775)